cypress/integration/networking.spec.ts

```diff
@@ -9,37 +9,56 @@ context("p2p networking", () => {
       "reacts to network state changes",
       { defaultCommandTimeout: 8000 },
       () => {
-        nodeManager.withTwoOnboardedNodes({}, (node1, node2) => {
-          nodeManager.asNode(node1);
-          commands.pick("connection-status-offline").should("exist");
-          nodeManager.connectTwoNodes(node1, node2);
-          commands.pick("connection-status-online").should("exist");
-          nodeManager.asNode(node2);
-          commands.pick("connection-status-online").should("exist");
-        });
+        nodeManager.withTwoOnboardedNodes(
+          { node1User: {}, node2User: {} },
+          (node1, node2) => {
+            nodeManager.asNode(node1);
+            commands.pick("connection-status-offline").should("exist");
+            nodeManager.connectTwoNodes(node1, node2);
+            commands.pick("connection-status-online").should("exist");
+            nodeManager.asNode(node2);
+            commands.pick("connection-status-online").should("exist");
+          }
+        );
       }
     );
   });
 
   it("replicates a project from one node to another", () => {
+    const maintainer = {
+      handle: "rudolfs",
+      fullName: "Rūdolfs Ošiņš",
+      passphrase: "1111",
+    };
+    const contributor = {
+      handle: "abbey",
+      fullName: "Abbey Titcomb",
+      passphrase: "2222",
+    };
+
     commands.withTempDir(tempDirPath => {
       nodeManager.withTwoOnboardedNodes(
-        { node1Handle: "rudolfs", node2Handle: "abbey" },
+        {
+          node1User: maintainer,
+          node2User: contributor,
+        },
         (maintainerNode, contributorNode) => {
           nodeManager.connectTwoNodes(maintainerNode, contributorNode);
           nodeManager.asNode(maintainerNode);
-          const newProjectPath = path.join(
+
+          const maintainerNodeWorkingDir = path.join(
             tempDirPath,
-            "maintainerNode/new-project"
+            "maintainerNode"
           );
-
-          cy.exec(`mkdir -p ${newProjectPath}`);
+          cy.exec(`mkdir -p ${maintainerNodeWorkingDir}`);
 
           ipcStub.getStubs().then(stubs => {
-            stubs.IPC_DIALOG_SHOWOPENDIALOG.returns(newProjectPath);
+            stubs.IPC_DIALOG_SHOWOPENDIALOG.returns(maintainerNodeWorkingDir);
           });
+          const projectName = "new-fancy-project.xyz";
+
           commands.pick("new-project-button").click();
-          commands.pick("name").type("new-fancy-project.xyz");
+          commands.pasteInto(["name"], projectName);
```

Comment:
I would suggest us undoing these changes from `pick(_).type(_)` to `pastInto(_, _)` for the sake of consistency and avoiding introducing another way of doing things that doesn't (seem to) offer any benefit.

Reply:
Ok, I see the note in the description saying that `pasteInto` speeds things up. I would still consider if that marginal speed gain is worth the introduction of inconsistency.